tests/robustness/snapif/snapif.go

```diff
@@ -0,0 +1,22 @@
+// Package snapif describes entities that are capable of performing
+// common snapshot operations
+package snapif
```

Comment:
better name for the package? `snap` , `snapshotter` but that stutters.

Reply:
went with snap